make/agents/svc/deploy.mk

```diff
@@ -22,7 +22,7 @@ deploy-rbac:
 .PHONY: deploy
 deploy: manifests kustomize ## Deploy controller to the K8s cluster specified in ~/.kube/config.
 	cd config/agents/svc && \
-		$(KUSTOMIZE) edit set image controller=${IMG} && \
+		$(KUSTOMIZE) edit set image agentsvc=${IMG} && \
```

Comment:
Nice catch! It seems to me we may need the same fix in Application Agent's makefiles and manifests.
Would it be a good idea to apply these changes in this PR or are we going to create a new one?